src/styles/main.scss

```
@@ -1,3 +1,246 @@
-body {
+@use './variables';
+
+.page {
+  height: 100vh;
   margin: 0;
+  display: grid;
+  grid-template-rows: 60px 50px auto 72px;
 }
+
+/* #region Page Header */
+
+.header {
+display: flex;
+justify-content: space-between;
+align-items: center;
+padding: 0 50px;
+grid-row: 1 / 2;
+box-sizing: border-box;
+box-shadow: 1px 2px 4px 1px rgba(0, 0, 0, 0.05);
+}
```

Need to fix code style everywhere
```suggestion
.header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 50px;
  grid-row: 1 / 2;
  box-sizing: border-box;
  box-shadow: 1px 2px 4px 1px rgba(0, 0, 0, 0.05);
}
```